recipes/libcufft/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "cufft" %}
+{% set version = "11.0.0.21" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/libcufft/{{ platform }}/libcufft-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 4650f79fcf377af410afedb56445f1332e8f213ec3c94d660f9a91328c3ecb4a  # [linux64]
+  sha256: e711671995e9d92d16015a4cd3751ec30710a49fa22f2b88a4a42905c248d861  # [ppc64le]
+  sha256: 91aa1ba5e68ca65778222c9a427a9214c4550d2f45e978e51779a207bafc17a3  # [aarch64]
+  sha256: a33f577d48ef432d5cfa8d0ebfa789cc5261af5dd84fd9c608235edc3f769e43  # [win]
+
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
```

Comment:
```suggestion
```

Reply:
In one place, I feel I squinted a sysroot being resolved to version 2.12 rather than 2.17 (I think).  But even if I didn't misinterpret that not sure if it adds up...

Reply:
> A subpackage does not automatically inherit any dependencies from its top-level recipe, so any build or run requirements needed by the subpackage must be explicitly specified.

Does this mean that _all_ requirements on the top level are moot?!  (maybe with the exception of the host requirements?